x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorTestUtils.java

```diff
@@ -197,7 +153,7 @@ private static FilteringValidation getRandomFilteringValidationError() {
     public static ConnectorFiltering getRandomConnectorFiltering() {
 
         Instant currentTimestamp = Instant.now();
-        int order = randomInt();
+        int order = randomShort();
```

Comment:
Why this change?

Reply:
++

Reply:
Similar issue to above, this is supposed to be `short` in the [Connector protocol](https://github.com/elastic/connectors/blob/main/docs/CONNECTOR_PROTOCOL.md#elasticsearch-mappings-for-elastic-connectors-sync-jobs). The introduction of system index mappings to unit tests brought this to light.